app/setup/styleguide.styles.js

```diff
@@ -32,6 +34,28 @@ module.exports = {
         fontSize: 20
       }
     },
+    Heading: {
+      heading1: {
```

Comment:
- [ ] The `:hover` is impacted by the global styles of the project. Would be nice if they are custom as well (just underline, also an issue on the zoom icons).